public/silent_renew.html

```diff
@@ -1,13 +1,20 @@
 <!DOCTYPE html>
-<html>
-  <head></head>
-  <body>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/oidc-client/1.10.1/oidc-client.min.js"></script>
-    <script>
-      var mgr = new Oidc.UserManager();
-      mgr.signinSilentCallback().catch((error) => {
-        console.error('silent_renew.html error', error);
-      });
-    </script>
-  </body>
-</html>
+<html lang="en">
+
+<head>
+  <title>Silent renewal</title>
+</head>
+
+<body>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/oidc-client-ts/2.2.2/browser/oidc-client-ts.min.js"
```

Comment:
As you're using ^3.0.1 of oidc-client-ts in package.json:
```json
 "oidc-client-ts": "^3.0.1",
```
then wouldn't it make sense to go with the same version here as well i.e. https://cdnjs.cloudflare.com/ajax/libs/oidc-client-ts/3.0.1/browser/oidc-client-ts.min.js

How did you get the integrity value for the script in the first place? I'm just thinking where or how to get the integrity value for the 3.0.1 version of the minified script, except by calculating the sha512 using some tool of course.

Reply:
The renew html is copied from the public UI, which is copied from the guide from HDS. Probably there is no difference, but sure it would make more sense to use the same version. I'll update it, even though the silent renew is not even in use.

Reply:
Upgraded now with a new integrity hash.